NanoServerPackage/NanoServerPackage.psm1

```diff
@@ -198,34 +187,30 @@ function Save-NanoServerPackage
 
         if($Path)
```

Comment:
$Path will be null in LiteralPath related parameter sets.

---
In reply to: [82316122](https://github.com/OneGet/NanoServerPackage/pull/6#discussion_r82316122) [](ancestors = 82316122)